.rtd-environment.yml

```diff
@@ -0,0 +1,7 @@
+name: rtd_sunpy_sphinx_theme
+channels:
+  - conda-forge
+dependencies:
+  - python=3.11
+  - pip
+  - graphviz!=2.42.*,!=2.43.*
```

Comment:
This is a more general question, but is this still a problem with the ubuntu packages?

Reply:
I didn't check, probably not now with 22.04.